modules/_labs/combobox/react/stories/stories.tsx

```diff
@@ -28,7 +27,7 @@ class Autocomplete extends React.Component<
 
   render() {
     const autocompleteResult = (textModifier: string) => (
-      <MenuItem onClick={action(`Clicked Result ${textModifier}`)}>
+      <MenuItem onClick={() => console.log(`Clicked Result ${textModifier}`)}>
```

Comment:
The `addon-action` import will essentially crash IE11 with a script lockup. I'm removing it here and replacing it with a `console.log`. It looks like the only reason this exists is to log out what happens so this should be the same functionality.

Reply:
I narrowed down this problem to how Storybook sends events. It uses post message to send messages. Sometimes multiple megabytes with each thing that happens inside Storybook (`action` is part of that). What is slowing things down is the `JSON.stringify` and another stringify function that is replacing strings. It can take even a 1000ms in Chrome occasionally.